src/getExtensionApi.ts

```diff
@@ -30,3 +31,15 @@ export async function getFunctionsApi(context: IActionContext): Promise<AzureFun
     // we still need to throw an error even if the user installs
     throw new UserCancelledError();
 }
+
+export async function getGitApi(): Promise<API> {
+    try {
+        const gitExtension: Extension<GitExtension> | undefined = extensions.getExtension<GitExtension>('vscode.git');
+        if (gitExtension) {
```

Comment:
Should probably do the same 'isActive' check done in `getFunctionsApi` (even if the git extension almost always activates before us - better to be safe). Might even be worth it to add a helper function that both `getGitApi` and `getFunctionsApi` since it seems like there's a bit of duplicate stuff